### PR TITLE
Test automation Integration with Eclipse pipeline #95

### PR DESCRIPTION
**Pull request by avsokol**

### CODE REVIEW

**Comment by zacanbrcom** [comment]
We should double check the TSPR for 3rd party dependencies used by the automation tool.

**Comment by avsokol** [comment]
Guys, can we push it somehow from the dead point? :)

**Comment by sergiuilie** [comment]
IP check process submitted

File: tests/theia_automation_lsp/inc/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Broadcom.
```
Comment (sergiuilie) [inline tests/theia_automation_lsp/inc/__init__.py]: same here

File: tests/theia_automation_lsp/robot_suites/__init__.robot [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Broadcom.
```
Comment (sergiuilie) [inline tests/theia_automation_lsp/robot_suites/__init__.robot]: delete if no need

File: tests/theia_automation_lsp/drivers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Broadcom.
```
[the remark this answers is not in the document]
Comment (avsokol) [reply tests/theia_automation_lsp/drivers/__init__.py]: Files like __init__ are needed to be able to import modules from other files

File: tests/theia_automation_lsp/inc/__init__.py (hunk 1 above)
Comment (avsokol) [reply tests/theia_automation_lsp/inc/__init__.py]: Files like __init__ are needed to be able to import modules from other files

File: tests/theia_automation_lsp/robot_suites/__init__.robot (hunk 2 above)
Comment (avsokol) [reply tests/theia_automation_lsp/robot_suites/__init__.robot]: Files like __init__.robot are needed to be able to run all robot scenarios by one command

File: tests/theia_automation_lsp/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+# Theia Automation
```
[the remark this answers is not in the document]
Comment (DStatWriter) [reply tests/theia_automation_lsp/README.md]: Changes made but generally good

File: tests/theia_automation_lsp/drivers/drv_config.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+import tarfile
+
+ENGINES_EXE = {
+    "chrome": "chromedriver",
+    "firefox": "geckodriver"
+}
+
+DRIVERS = {
+    "darwin": {
+        "firefox": "https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-macos.tar.gz",
```
Comment (sergiuilie) [inline tests/theia_automation_lsp/drivers/drv_config.py]: please make a list with the dependencies and versions use for the drivers in order to submit it for IP checks

Comment (avsokol) [reply tests/theia_automation_lsp/drivers/drv_config.py]: name: geckodriver
url: https://github.com/mozilla/geckodriver
license: Mozilla Public License 2.0
version: v0.24.0

name: chromedriver
url: https://chromedriver.chromium.org/
license: BSD license
version: 77.0.3865.40

Comment (avsokol) [reply tests/theia_automation_lsp/drivers/drv_config.py]: Some more details:
gecko:
Mozilla Public License 2.0
https://www.mozilla.org/en-US/MPL/2.0/
download from: https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-win64.zip


chrome:
https://chromedriver.chromium.org/
BSD license
https://cs.chromium.org/chromium/src/LICENSE
download from: https://chromedriver.storage.googleapis.com/77.0.3865.40/chromedriver_win32.zip

Comment (avsokol) [reply tests/theia_automation_lsp/drivers/drv_config.py]: Cannot update/create cq as I'm not committer AFAIU

Comment (sergiuilie) [reply tests/theia_automation_lsp/drivers/drv_config.py]: I will create the request

Comment (sergiuilie) [reply tests/theia_automation_lsp/drivers/drv_config.py]: status: waiting for approval

**Pull request merged**